src/kernel/src/fs/vnode.rs

```diff
@@ -53,6 +57,7 @@ impl Drop for Vnode {
 #[derive(Debug)]
 pub enum VnodeType {
     Directory(bool),
+    Bad,
```

Comment:
Making something in an invalid state is not work well with Rust. So let's remove this.

Reply:
So... how do we represent this state? Option<Vnode>?

Reply:
The vnode itself should never be in that state for us. That mean it does not exists.

Reply:
Oh